cypress/integration/google.spec.ts

```diff
@@ -3,4 +3,5 @@ describe("This is my first cypress test", () => {
         cy.visit("https://www.google.com/");
         cy.title().should("eq", "Google");
     });
-});
\ No newline at end of file
+}); 
+    
```

Comment:
Line 7 has some whitespaces, delete those whitespaces or tabs, without deleting line 7. Or maybe you have two blank lines, and it is just needed one.